codespell_lib/data/dictionary.txt

```diff
@@ -5807,13 +5831,19 @@ inablility->inability
 inacccessible->inaccessible
 inaccesible->inaccessible
 inaccessable->inaccessible
+inactivate->inactive
```

Comment:
This one is an acceptable verb, so it should stay the first suggestion. You can also suggest "deactivate".